dev_guide/managing_images.adoc

```diff
@@ -777,3 +777,68 @@ NAME       DOCKER REPO                        TAGS      UPDATED
 my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
 ----
 ====
+[[writing-image-streams-for-s2i-builders]]
+== Writing Image Streams for S2I Builders
```

Comment:
@vikram-redhat Looks like this section got added accidentally while trying to pick https://github.com/openshift/openshift-docs/pull/3466 to enterprise-3.2. The actual PR this came from (https://github.com/openshift/openshift-docs/pull/3335) doesn't list enterprise-3.2 as a label so I'm gonna yoink this from 3.2.